c_glib/arrow-glib/input-stream.cpp

```diff
@@ -325,6 +325,30 @@ garrow_seekable_input_stream_read_at(GArrowSeekableInputStream *input_stream,
 }
 
 
+/**
+ * garrow_seekable_input_stream_peek:
+ * @input_stream: A #GArrowSeekableInputStream.
+ * @n_bytes: The number of bytes to be peeked.
+ *
+ * Returns: (transfer full): The string representation of any buffered bytes,
+ *   up to the indicated number.
+ *
+ *   It should be freed with g_free() when no longer needed.
+ *
+ * Since: 0.12.0
+ */
+const gchar *
```

Comment:
Could you remove `const`? We should not use `const` for objects that are needed to be freed.
If we remove `const`, we will be able to remove `(transfer full)`. Because GObject Introspection guesses the transfer value.

Reply:
Oh... I'll remove `const`.